src/SearchParameters/index.js

```diff
@@ -1393,20 +1413,6 @@ SearchParameters.prototype = {
   filter: function(filters) {
     return filterState(this, filters);
   },
-  /**
-   * Helper function to make it easier to build new instances from a mutating
-   * function
-   * @private
-   * @param {function} fn newMutableState -> previousState -> () function that will
-   * change the value of the newMutable to the desired state
-   * @return {SearchParameters} a new instance with the specified modifications applied
-   */
-  mutateMe: function mutateMe(fn) {
```

Comment:
For reference, this is only used in [`connectConfigure` in InstantSearch.js](https://github.com/algolia/instantsearch.js/blob/49a5b64e42deec3453669d7dce788395cf898ba8/src/connectors/configure/connectConfigure.js#L96-L99). One solution for this removal could be to set the `widgetParams` search parameters to `undefined`.